pkgs/by-name/li/libnvidia-container/package.nix

```diff
@@ -103,10 +125,14 @@ stdenv.mkDerivation rec {
   postInstall =
     let
       inherit (addDriverRunpath) driverLink;
-      libraryPath = lib.makeLibraryPath [ "$out" driverLink "${driverLink}-32" ];
+      libraryPath = lib.makeLibraryPath [
+        "$out"
+        driverLink
+        "${driverLink}-32"
```

Comment:
Note on legacy: as noted above, on 32-bit `hostPlatform`s this would evaluate into `/run/opengl-driver-32-32`